lightning/src/routing/router.rs

```diff
@@ -143,12 +143,16 @@ struct RouteGraphNode {
 	// - how much is needed for a path being constructed
 	// - how much value can channels following this node (up to the destination) can contribute,
 	//   considering their capacity and fees
-	value_contribution_msat: u64
+	value_contribution_msat: u64,
+	/// The maximum htlc_minimum_msat along the path, taking into consideration the fees required
+	/// to meet the minimum over the hops required to get there.
```

Comment:
I think this subtlety around `htlc_minimum_msat` + fees to cover such minimum should be explained better. E.g "The *effective* htlc_minimum_msat required by a hop to forward a payment must also includes fees. `effective_htlc_minimum_msat` = `htlc_minimum_msat` + `fee_base_msat` + (`htlc_minimum_msat` * `fee_proportional_millionths`)